apps/src/templates/projects/ProjectAppTypeArea.jsx

```diff
@@ -133,7 +133,7 @@ class ProjectAppTypeArea extends React.Component {
    * @returns {$.Deferred} Deferred object after the network request has
    *   completed and the done handler has been run (if successful).
    */
-  fetchOlderProjects() {
+  fetchOlderProjects = () => {
```

Comment:
I suspect this binding is unnecessary because the function that's calling it already has `this` bound correctly, but I'm not sure. @islemaster may know. Better safe than sorry, so fine to keep this the way you have it in this PR for now.

Reply:
That's correct.  It's not really harmful to bind it here, but you don't need to bind functions that are:

- React lifecycle methods (unless you call them manually at some point)
- Only called with context, i.e. `this.methodName()`